edenai_apis/apis/minimax/minimax_api.py

```
@@ -0,0 +1,268 @@
+        self.api_settings = load_provider(
+            ProviderDataEnum.KEY, self.provider_name, api_keys=api_keys
+        )
+        self.base_url = "https://api.minimax.io/v1"
+        self.api_key = self.api_settings.get("api_key")
+        self.group_id = self.api_settings.get("group_id")
+        self.client = OpenAI(
+            api_key=self.api_key,
+            base_url=self.base_url,
+        )
+
+    def video__generation_async__launch_job(
```

Pylint (3.3.7)

[refactor] Too many arguments (10/7)

(R0913)